### ISSUE TITLE
Download metadata button misleading

### ISSUE THREAD

**Issue by jbpoline** [CONP-PCNO/conp-portal#227]
It is too easy to think you are downloading the data when clicking on the big download arrow (As it is visually proeminent) - I would swap behaviour with click on title

**Comment by cmadjar** [comment]
Should make it clearer that this download button is downloading the overall dataset metadata, not the data or files metadata.

**Comment by glatard** [comment]
It could be just a smaller and different icon, for instance one that looks like [this](https://www.iconfinder.com/icons/1180472/brackets_curly_file_paper_sheet_icon).

**Comment by glatard** [comment]
at the meeting @jbpoline also suggested that this button is added to the data description page.

**Comment by jbpoline** [comment]
yes, the metadata download would be on the dataset description (possibly with a visualizer in the future?)

**Comment by cmadjar** [comment]
Removed the discussion required as I think we came to a consensus which is:
1) use [the following icon](https://www.iconfinder.com/icons/1180472/brackets_curly_file_paper_sheet_icon) from @glatard's suggestion above (or an equivalent icon)
2) remove the Download metadata button from the dataset list page altogether as misleading
3) replace the Download metadata icon by the metadata icon from 1) on the dataset page

Later on, we can create a visualizer for the dataset description. Great idea @jbpoline ! I will create a separate issue for that one as this is a task on its own.

Adding the high priority flag as this would be good if it could be done before next week's webinar.
